src/shared/components/plots/downsampleByGrouping.spec.ts

```diff
@@ -31,12 +31,18 @@ describe('downsampleByGrouping', () => {
             {
                 x: 0,
                 y: 0,
-                data: [{ x: 0, y: 0, blah: 'A' }, { x: 1, y: 0, blah: 'B' }],
```

Comment:
https://prettier.io/blog/2019/11/09/1.19.0.html#break-arrays-of-arraysobjects-if-each-element-has-more-than-one-elementproperty-6694-by-sosukesuzuki